sponsorship_compassion/model/contracts.py

```diff
@@ -15,7 +15,7 @@
 from openerp.tools import DEFAULT_SERVER_DATE_FORMAT as DF
 
 from datetime import datetime
-from dateutil.relativedelta import relativedelta
+from dateutil.relativedelta import relativedelta

```

Comment:
Please ensure that each contributor has a well configured development tool. It's not visible on github, but the difference between theses two line is an extra CR (^m) character at end of line. This is typical from windows.